go/api/set_commands.go

```diff
@@ -278,4 +278,27 @@ type SetCommands interface {
 	//
 	// [valkey.io]: https://valkey.io/commands/spop/
 	SPop(key string) (Result[string], error)
+
+	// SMIsMember returns whether each member is a member of the set stored at key.
+	//
+	// See [valkey.io] for details.
+	//
+	// Parameters:
+	//   key - The key of the set.
+	//
+	// Return value:
+	//   A []Result[bool] containing whether each member is a member of the set stored at key.
+	//
+	// Example:
+	//	 client.SAdd("myKey", []string{"one", "two"})
+	//   value1, err := client.SMIsMember("myKey", []string{"two", "three"})
+	//   // value1[0].Value() == true
```

Comment:
Other commands have another format of showing example response. I don't mind against this format, but we need to discuss and align to have one common format eventually.
```suggestion
	//   value1[0].Value(): true
```